client/app/account/user/dashboard/translations/Messages_fr_FR.json

```
@@ -0,0 +1,18 @@
+{
+    "user_account_dashboard_shortcut":"Raccourcis",
+    "user_account_dashboard_shortcut_ALL_BILLS":"Voir mes factures",
+    "user_account_dashboard_shortcut_PAYMENT_FOLLOW_UP":"Suivre mes paiements",
+    "user_account_dashboard_shortcut_ADD_PAYMENT_METHOD":"Ajouter un moyen de paiement",
+    "user_account_dashboard_shortcut_MANAGE_SERVICES":"Gérer mes services",
+    "user_account_dashboard_shortcut_ADD_CONTACT":"Ajouter un contact",
+    
+    "user_account_dashboard_last_bill":"Ma dernière facture",
+    "user_account_dashboard_last_bill_reference":"Référence",
+    "user_account_dashboard_last_bill_date":"Date",
+    "user_account_dashboard_last_bill_amount":"Montant",
+    "user_account_dashboard_last_bill_see":"Voir ma facture",
+    
+    "user_account_dashboard_profile":"Mon profil",
+    "user_account_dashboard_profile_edit":"Editer mon profil",
+    "user_account_dashboard_profile_support_level":"Mon niveau de support : "
```

```suggestion
    "user_account_dashboard_profile_support_level":"Mon niveau de support :"
```
nbsp + remove trailing whitespace :)